roles/rhods/rhods_notebook_locust_scale_test/meta/main.yml

```
@@ -1,3 +1,3 @@
 ---
-dependencies:
+dependencies: []
   #- role: check_deps
```

I guess the linter spot that. Could you rather change it this way?
```suggestion
dependencies: 
  - role: check_deps
```
it shouldn't have been committed with the `check_deps` dependency missing.

---

Sure thing, I believed this was like that in purpose.